pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md

```diff
@@ -12,6 +12,8 @@ The following table describes the list of applications that can be deployed to a
 Review the [project application service resource requirements](./platform-application-requirements/) to ensure that the attached clusters have sufficient resources.
 -->
 
+When deploying and upgrading applications, platform applications come as a bundle; they are tested as a single unit and you must deploy or upgrade them in a single process, for each workspace. This means all clusters in a workspace have the same set and version of platform applications deployed.
+
 ### Deploy applications in a project
```

Comment:
Shouldn't this page have the instructions on how to upgrade the platform applications in a work space?

Reply:
We do support upgrading platform apps per workspace - for the management cluster/workspace, this is automatically done as part of the kommander upgrade process. But for other workspaces, there will be a separate CLI command to upgrade them. We'll need an upgrade page under workspaces/ in docs as well, we can point to that from here. All project platform apps will automatically get upgraded with the workspace

Reply:
@gracedo - Regarding "wll be a CLI command", is this being done in v2.2, or later?

Reply:
2.2! it's in progress :)